industrial_ci/src/tests/source_tests.sh

```diff
@@ -103,6 +109,8 @@ if [ "${USE_MOCKUP// }" != "" ]; then
     ln -s "$TARGET_REPO_PATH/$USE_MOCKUP" $CATKIN_WORKSPACE/src
 fi
 
+catkin config --install
```

Comment:
https://github.com/ros-industrial/industrial_ci/pull/150#pullrequestreview-31649321
>>    The current behaviour is really odd:
>>
>>    1.    build to build/devel space
>>    1.    run build test
>>    1.    clean
>>    1.    build to build/devel space and install to install space
>>    1.    run rostest on install space
>>
>>    So in step 4 we are rebuilding everything from step 1, which doubles the time the build takes.
>
> Ok, with this PR we're skipping the build in 4, is that correct?
> What I was afraid was to skipping step 1 and 2 (and 3 obvisously), which could cause errors on the repos where no install rules are not defined. But looks like I was wrong and worrying about nothing.

I overlooked adding this line. With this `NOT_TEST_INSTALL=true` that allows users to avoid testing against `install` space becomes not working (e.g. https://github.com/ros-industrial-consortium/descartes/pull/198#issuecomment-301516448).